### ISSUE TITLE
ALL - generic improvements to our web presence

### ISSUE THREAD

**Issue by despiegk** [threefoldfoundation/home#123]
its the result of a generic check, not sure where it all belongs

(sam added some tags here to make clearer)

-  the main page green should become another colour, maybe blue, green is not a nice colour for impact website (**NOTE: We decided not to move forward with this right now because we use blue, green, and purple intentionally - see [here](http://www.arttherapyblog.com/online/color-psychology-psychologica-effects-of-colors/#coolcolors) – we can discuss for future sprints if necessary**)

- [x] menus and footers for all websites
- - [x] farming
- - [x] cloud
- - [x] marketplace
- - [x] main

### ACI

- [x] the people should be able to search and see them in categories (@Vilnite)
- [x] the projects should be able to search and see them in categories (@Vilnite)
- **MOVED TO NEXT SPRINT IN NEW ISSUE:** there are quite some people missing e.g. Michel Coucke, we need to go over everyone who is relevant for our project and they need to be there (e.g. people from this [deck](https://docs.google.com/presentation/d/1tRPjubgQaspXIXFV1vh8eiyBGrXAW6yuIeZbMN7a9qA/edit#slide=id.gc6f21bb149_0_806))

### Cloud

- [x] link to wiki (and specific places on wiki for each of the product sections) – see above
- [x] explain the pricing somewhere, with a typical pricing page (pricing is on foundation wiki now)
- - [x] @samtaggart figure out which component can be used
- - [x] @samtaggart get any image needed executed with Zia
- - [x] @samtaggart work with @Vilnite to get it live 

### Marketplace

- [x] https://marketplace.threefold.io/about needs to be the home page, much more powerful
- - [x] @samtaggart look to see what the implications here, if anything needs to be reworded
- - [x] @samtaggart work with @Vilnite to execute

**Comment by samtaggart** [comment]
@Sacha96 If you can help on some aspects of this issue today that would be ideal. Let's discuss.

**Comment by Sacha96** [comment]
Menus and footers edited appart from aci, tech and digital twin.

**ACI** 
- no content for footer but will create footer for legal: terms & conditions, privacy policy etc... beginning next spring 

**tech**
-  wiki is still private, but same as aci need legal in - where should we lead for this?
@VonSub can you guide me to right links

**Digital twin**
- waiting on last update of website and wiki updates, will be done beginning next sprint, will compile most important links to learn about DT for footer and website.
@AlexandreHannelas can you help compile?

**Comment by samtaggart** [comment]
Issue created around pricing : https://github.com/threefoldfoundation/www_threefold_cloud/issues/18
